src/main/kotlin/screeps/api/structures/StructureSpawn.kt

```diff
@@ -5,18 +5,16 @@ import screeps.api.*
 abstract external class StructureSpawn : Structure, Owned, SpawnEnergyProvider {
     val memory: SpawnMemory
     val name: String
-
     val spawning: Spawning?
 
     /**
      * Start the creep spawning process. The required energy amount can be withdrawn from all spawns and utils in the room.
      */
-    fun spawnCreep(body: Array<BodyPartConstant>, name: String): ScreepsReturnCode
-
-    /**
-     * Start the creep spawning process. The required energy amount can be withdrawn from all spawns and utils in the room.
-     */
-    fun spawnCreep(body: Array<BodyPartConstant>, name: String, opts: SpawnOptions): ScreepsReturnCode
+    fun spawnCreep(
+        body: Array<BodyPartConstant>,
+        name: String,
+        opts: SpawnOptions = definedExternally
```

Comment:
IIRC this will crash

Reply:
Looks to be fine in the transpiled Javascript.